js/main.js

```diff
@@ -23,5 +46,60 @@ function handleSubmit(event) {
   data.nextEntryId++;
   data.entries.unshift(newEntry);
   $img.setAttribute('src', './images/placeholder-image-square.jpg');
+
+  showEntries(event);
   $form.reset();
 }
+
+function showNewForm(event) {
+  $views.forEach(node => {
+    if (node.getAttribute('data-view') === 'entry-form') {
+      node.className = 'view';
+    } else {
+      node.className = 'view hidden';
+    }
+  });
+  data.view = 'entry-form';
+}
+
+function showEntries(event) {
+  $views.forEach(node => {
+    if (node.getAttribute('data-view') === 'entries') {
+      node.className = 'view';
+    } else {
+      node.className = 'view hidden';
+    }
+  });
+  data.view = 'entries';
+}
```

Comment:
You can likely turn this into one function that receives an argument of a string which is the name of the view to show